src/strings/en-us.json

```diff
@@ -1391,6 +1392,7 @@
     "Season": "Season",
     "SecondarySubtitles": "Secondary Subtitles",
     "SelectAdminUsername": "Please select a username for the admin account.",
+    "SelectPreferredTranscodeVideoCodecHelp": "Select the preferred video codec to transcode to. If the preferred codec is not supported, the server will use the next best available codec.",
```

Comment:
```suggestion
    "SelectPreferredTranscodeVideoCodecHelp": "Select the preferred video codec for transcoding. If the preferred codec is not supported, the server will use the next best available codec.",
```

Reply:
I personally think that it is more obvious what it means the way I stated it, but I'm happy to change it. My way is also the same as the string for the preferred audio codec.